<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Mukilsai" failures="1" tests="24" name="requests.PostRequest" time="7.670" errors="0" timestamp="2024-10-18T15:13:35 EDT" skipped="0">
  <testcase classname="requests.PostRequest" name="Post_New_user" time="0.268"/>
  <testcase classname="requests.PostRequest" name="Post_New_user" time="0.267"/>
  <testcase classname="requests.PostRequest" name="Post_New_user" time="0.276"/>
  <testcase classname="requests.PostRequest" name="Post_New_user" time="0.274"/>
  <testcase classname="requests.PostRequest" name="Post_New_user" time="0.390"/>
  <testcase classname="requests.PostRequest" name="Post_New_user" time="0.262"/>
  <testcase classname="requests.PostRequest" name="Post_New_user" time="0.271"/>
  <testcase classname="requests.PostRequest" name="Post_New_user" time="1.076"/>
  <testcase classname="requests.PostRequest" name="Post_New_user" time="0.269"/>
  <testcase classname="requests.PostRequest" name="Post_New_user" time="0.280"/>
  <testcase classname="requests.PostRequest" name="Post_New_user" time="0.254"/>
  <testcase classname="requests.PostRequest" name="Post_New_user" time="0.288"/>
  <testcase classname="requests.PostRequest" name="Post_New_user" time="0.272"/>
  <testcase classname="requests.PostRequest" name="Post_New_user" time="0.269"/>
  <testcase classname="requests.PostRequest" name="Post_New_user" time="0.266"/>
  <testcase classname="requests.PostRequest" name="Post_New_user" time="0.275"/>
  <testcase classname="requests.PostRequest" name="Post_New_user" time="0.287"/>
  <testcase classname="requests.PostRequest" name="Post_New_user" time="0.273"/>
  <testcase classname="requests.PostRequest" name="Post_New_user" time="0.269"/>
  <testcase classname="requests.PostRequest" name="Post_New_user" time="0.248">
    <failure message="Expected status code is 409 Actual status is : 201 expected [409] but found [201]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Expected status code is 409 Actual status is : 201 expected [409] but found [201]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:1419)
at org.testng.Assert.assertEquals(Assert.java:1383)
at requests.PostRequest.Post_New_user(PostRequest.java:44)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- Post_New_user -->
  <testcase classname="requests.PostRequest" name="Post_New_user" time="0.270"/>
  <testcase classname="requests.PostRequest" name="Post_New_user" time="0.295"/>
  <testcase classname="requests.PostRequest" name="Post_New_user" time="0.267"/>
  <testcase classname="requests.PostRequest" name="Post_New_user" time="0.504"/>
</testsuite> <!-- requests.PostRequest -->
